crates/api_common/src/utils.rs

```diff
@@ -668,10 +669,14 @@ pub async fn purge_image_posts_for_community(
 }
 
 pub async fn remove_user_data(
+  mod_person_id: PersonId,
   banned_person_id: PersonId,
+  reason: &Option<String>,
```

Comment:
Probably better to use `Option<&str>` instead of `&Option<String>`.

Reply:
We used to do that actually, but it ended up being pretty pointless. Either one works, but `&Option` requires less transformations and is a little less verbose.